sdks/go/pkg/beam/core/runtime/coderx/string.go

```diff
@@ -23,6 +23,8 @@ import (
 
 // NewString returns a coder for the string type. It uses the native
 // []byte to string conversion.
+//
+// Only for custom coder test use.
 func NewString() (*coder.CustomCoder, error) {
```

Comment:
If this is only meant to be used for custom coder tests, should it just be renamed instead? It seems unlikely to be used by user code right now.

Reply:
Note that most things that aren't in the main `beam` package are really intended for user use.  I'll send a separate PR moving/removing this code, I just didn't want this PR to become too messy.